template-test/ci-run.sh

```diff
@@ -13,7 +13,8 @@ APP_NAME="${VARIANT}-test-app"
 rm -rf template-test/dummy/$APP_NAME
 cd template-test/dummy
 
-echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new $APP_NAME -d postgresql -m $TEMPLATE
+echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new $APP_NAME -d postgresql -m $TEMPLATE \
+--skip_javascript
```

Comment:
```suggestion
echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new $APP_NAME -d postgresql -m $TEMPLATE \
  --skip_javascript
```

🐘 just so it looks more like it's a continue on from the previous line when scanning.

Reply:
ok fixed this :) thanks